lib/matplotlib/widgets.py

```diff
@@ -2264,8 +2282,22 @@ def _release(self, event):
         # self._pressv is deprecated but we still need to maintain it
         self._pressv = None
 
+        self._active_handle = None
+
         return False
 
+    def _hover(self, event):
+        """Update the canvas cursor if it's over a handle."""
+        if self.ignore(event):
+            return
+
+        if self._active_handle is not None:
+            # Do nothing if button is pressed and a handle is active.
```

Comment:
I'm not really sure what this case is supposed to handle?

Reply:
When you're dragging, the cursor stays on the resizer. If you're dragging and move close enough to an edge before the selector moves, then it could trigger the hover effect and _disable_ the resize cursor even though the selector is still active. I think this may only apply if you are dragging from the centre.

Reply:
Ah, I see, perhaps just add a comment to note that this is to handle `draw_from_anywhere=True`?  (which I had missed)
I can't repro it locally but I guess that depends on mouse latency.

Reply:
Yes, I'm using X forwarding (though still on the local network), so there may be some delay between events.